web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -688,6 +688,19 @@ const UpstreamForm: React.FC<Props> = forwardRef(
                 })}
               </Select>
             </Form.Item>
+            <Form.Item
+              label={formatMessage({ id: 'page.upstream.step.scheme' })}
+              name="scheme"
+            >
+              <Select disabled={readonly}>
+              {['http', 'https', 'grpc', 'grpcs'].map((item, index) => (
+                <Select.Option value={item} key={index}>
```

Comment:
In computer science, an associative array, map, symbol table, or dictionary is an abstract data type composed of a collection of (key, value) pairs, such that each possible key appears at most once in the collection

https://en.wikipedia.org/wiki/Associative_array

So my view is that `key` should come before `value` and also based on order alphabetically.

```suggestion
                <Select.Option key={index} value={item}>
```

Reply:
That's fine, and not urgent 👍